std/traits.d

```diff
@@ -164,7 +164,11 @@
  */
 module std.traits;
 
-import std.typetuple; // TypeTuple
+import std.meta : AliasSeq, allSatisfy;
+
+// Legacy inheritance from std.typetuple - should be deprecated
+// See also: https://github.com/dlang/phobos/pull/5484#discussion_r122602797
+public import std.meta : staticMap;
```

Comment:
I will add a `deprecation` message as soon as we have figured out if just `staticMap` or more symbols  are affected.